make/go/base.mk

```diff
@@ -193,7 +193,7 @@ generate: ## Run all generation steps.
 
 .PHONY: checknodiffgenerated
 checknodiffgenerated:
-	@ if [ -d .git ]; then \
+	@ if [[ -d .git || -f .git ]]; then \
```

Comment:
Is this being pulled from makego?

Reply:
Yes, this PR https://github.com/bufbuild/makego/pull/96